.travis.yml

```diff
@@ -39,3 +39,7 @@ script:
   - sbt -J-Xms1g -J-Xmx5g ++$TRAVIS_SCALA_VERSION daffodil-japi/genjavadoc:doc
   - sbt -J-Xms1g -J-Xmx5g ++$TRAVIS_SCALA_VERSION daffodil-sapi/doc
   - sbt -J-Xms1g -J-Xmx5g ++$TRAVIS_SCALA_VERSION ratCheck
+  - sbt clean coverage test it:test
```

Comment:
I'll add the -J options and make it so we only run the tests once.

As for needing coverageAggregate, I'm not sure since I haven't used codecov before.  I know the coverageAggregate is what made the nice index.html file with links to all the subproject coverage results.  It doesn't take long to run (just couple of seconds on my system) so my vote would be to leave it in for now and see how things look on codecov.